src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -420,6 +422,45 @@ class AccountControllerTest @Autowired constructor(
         }
     }
 
+    @NestedTest
+    @DisplayName("POST /accounts/changePassword/{id}")
+    inner class ChangePassword {
+        private val password = "test_password"
+        private val changePasswordAccount: Account = ObjectMapper().readValue(
+            ObjectMapper().writeValueAsString(testAccount),
+            Account::class.java
+        )
+
+        @BeforeEach
+        fun addAccount() {
+            changePasswordAccount.password = encoder.encode(changePasswordAccount.password)
+            repository.save(changePasswordAccount)
+        }
+
+        @Test
+        fun `should change password`() {
```

Comment:
You should also test the sad path, ie. the provided password is wrong and the request should fail.